### ISSUE TITLE
Update Site broken

### ISSUE THREAD

**Issue by D023535** [sandbox-wildwebdeveloper/wildwebdeveloper#73]
Hi, 

the mentioned link to the market place 
https://marketplace.eclipse.org/content/wildwebdeveloper-web-development-eclipse-ide-experimental

and the update site 

http://repository.jboss.org/nexus/content/unzip/unzip/org/eclipse/wildwebdeveloper/repository/0.1.0-SNAPSHOT/repository-0.1.0-SNAPSHOT.zip-unzip/

are not working currently. 
thank you

**Comment by mickaelistria** [comment]
Thanks a lot for the report.

> the mentioned link to the market place https://marketplace.eclipse.org/content/wildwebdeveloper-web-development-eclipse-ide-experimental

I just fixed it.

> and the update site http://repository.jboss.org/nexus/content/unzip/unzip/org/eclipse/wildwebdeveloper/repository/0.1.0-SNAPSHOT/repository-0.1.0-SNAPSHOT.zip-unzip/

It seems like that travis configuration is erroneous and fails to perform the deploy step. I've deployed/published manually build from last commit to this URL.
When Wild Web Developer moves to Eclipse.org (within a few days), I'll update the infrastructure to use Jenkins instance at Eclipse.org which should work better than the Travis/repository.jboss.org combo. So I don't think it's worth spending more effort in trying to fix the travis scripts right now if it's to drop it next week.

So I guess we can consider this bug as resolved ;)
